libs/core/src/lib/input-group/input-group.component.html

```diff
@@ -28,13 +28,14 @@
                 type="button"
                 fdType="transparent"
                 [compact]="compact"
+                [id]="_addOnButtonId"
                 [attr.tabindex]="buttonFocusable ? 0 : -1"
-                [attr.title]="iconTitle"
+                [attr.title]="iconTitle ? iconTitle : glyph ? glyph : addOnText"
                 [glyph]="glyph"
                 [class.fd-shellbar__button]="inShellbar"
                 [class.is-expanded]="isExpanded"
-                [attr.aria-expanded]="isExpanded"
-                [attr.aria-label]="glyphAriaLabel"
+                [attr.aria-expanded]="inShellbar ? isExpanded : null"
+                [attr.aria-label]="glyphAriaLabel ? glyphAriaLabel : glyph ? glyph : addOnText"
```

Comment:
Here could be used parentheses for better readability:
`[attr.aria-label]="glyphAriaLabel ? glyphAriaLabel : (glyph ? glyph : addOnText)"`

Reply:
made changes as suggested